pkg/controller/nexus/resource/resources.go

```diff
@@ -37,7 +38,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Isn't this the same as not renaming the import? Also, while we're at it, `v1`, `v1beta1` and so on are very cryptic imo. I think we should always use the package name and version when renaming, like `appsv1` above. Wdyt?

```suggestion
	corev1 "k8s.io/api/core/v1"
```

This would require changing its usages below as well, of course.

Reply:
I agree, but that was changed by the lint/IDE itself. I will take this into account when refactoring/implementing new features to the code in the future. Let's leave that as is for now.